library/tasks/HISAT2.wdl

```diff
@@ -120,6 +124,7 @@ task HISAT2RSEM {
     cpu: "(optional) the number of cpus to provision for this task"
     disk: "(optional) the amount of disk space (GB) to provision for this task"
     preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
+    max_retries: "(optional) retry this number of times if task fails"
```

Comment:
It would be great to add some context from your conversation with rex above -- a caution that this is basically only to catch one-off errors that occur at production scale would be helpful for users.